packages/dapp/src/components/Switch.tsx

```diff
@@ -0,0 +1,16 @@
+import { CheckBox } from 'grommet';
+
+export interface SwitchProps {
```

Comment:
Please check is Grommet has its own typings. If yes, please use them instead of custom.